src/reuse/header.py

```diff
@@ -144,6 +144,86 @@ def create_header(
     return new_header
 
 
+def comment_has_spdx(comment: str) -> bool:
+    """Check whether comment already contains SPDX info."""
+    try:
+        existing_spdx = extract_spdx_info(comment)
+    except (ExpressionError, ParseError):
+        existing_spdx = (None, None)
+    return existing_spdx[0] or existing_spdx[1]
```

Comment:
I don’t think it works.

Reply:
```
>>> bool( (None, None) )
True
```

Reply:
Ah… `any(existing_spdx)` should work instead.

Reply:
Oh, ok, then it explains why you had this in your code :). To be honest I would prefer to leave it my way... because I assumed that this `any` that you had was a mistake 😆.